ch/proc version build url 1gmdpak

hoffmanc opened the pull request.

hoffmanc committed:
add test for missing serial number #1gmdn3y

hoffmanc committed:
attempt to create activity when missing serial number #attempt

hoffmanc committed:
add proc version and build url to work orders table #1gmdpak

zacnelson commented:
![image](https://user-images.githubusercontent.com/2513422/135810338-804617dc-5b6b-4bcd-b269-85fc4b25b73b.png)

Currently seeing this when clicking on a completed Pico Work Order. I think the build URL needs more line wrapping as it's currently pushing the first column (maybe?) too far to the right.

zacnelson commented:
> Currently seeing this when clicking on a completed Pico Work Order. I think the build URL needs more line wrapping as it's currently pushing the first column (maybe?) too far to the right.

Not sure how to control what shows up in this pop-up. Other than that, this is great.

hoffmanc committed:
use named volume for pg data and auto-reload views and module code in dev #1gmdpak

named volume is because the dev options were causing odoo to complain
about file permissions inside the data/postgres directory.  There's
probably a better solution, but his works for me :)

hoffmanc committed:
conditionally display build url button #1gmdpak

hoffmanc committed:
add fields to detail view for WO #1gmdpak

hoffmanc committed:
rename setter field #1gmdpak

hoffmanc committed:
account for > 1 wo per mo #1gmdpak

zacnelson reviewed (approved):
Functionally working for me.

sfenig commented on pico_mrp/data/activity_data.xml:
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
+    <record id="mail_activity_type_missing_produce_serial" model="mail.activity.type">
```
is the activity stuff working?  thought you couldn't get that sorted?

hoffmanc replied:
no I couldn't get it working - I will remove this

sfenig reviewed (approved):
Nice man.  Thanks for cleaning this up and adding some more scripts/docs.  I thought the activity stuff ended up being a no go.    Did you get that working?  Can we get it going generically for the unknown error case as well, then?

hoffmanc committed:
remove activity attempt #1gmdpak
